common/pipeline_storage/src/test/scala/uk/ac/wellcome/pipeline_storage/fixtures/ElasticIndexerFixtures.scala

```diff
@@ -52,4 +52,13 @@ trait ElasticIndexerFixtures extends ElasticsearchFixtures with Akka {
   implicit def canonicalId[T](
     implicit indexable: Indexable[T]): CanonicalId[T] =
     (doc: T) => indexable.id(doc)
+
+  def ingestInOrder[T](indexer: ElasticIndexer[T])(documents: T*)(
+    implicit ec: ExecutionContext): Future[Either[Seq[T], Seq[T]]] =
+    documents.tail.foldLeft(indexer.index(List(documents.head))) {
+      (future, doc) =>
+        future.flatMap { _ =>
+          indexer.index(List(doc))
+        }
+    }
```

Comment:
✨

Reply:
Minor: usually our tests just import the global ExecutionContext at the call site rather than taking it implicitly, although it's not a big deal.

Reply:
It got upset when I tried to do that for some reason 🤷‍♂️